spec/04-basic-declarations-and-definitions.md

```diff
@@ -840,6 +840,23 @@ class C extends I {
 Here, it is OK to leave out the result type of `factorial`
 in `C`, even though the method is recursive.
 
+### Tail-call Elimination
```

Comment:
```suggestion
### Tail-Recursive Call Elimination
```
We don't have general tail call elimination

Reply:
The text says as much, but since hyphenization and capitalization are also sources of controversy, I take the recommended edit without further ado.